templates/scripts.js

```diff
@@ -0,0 +1,18 @@
+document.getElementById('url-form').addEventListener('submit', async function(event) {
```

Comment:
Move to `templates/static`

Reply:
Thank you for the feedback! I will move the `scripts.js` file to the `templates/static` directory and update the paths accordingly.

I'll make the necessary changes and update the PR.